moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -743,10 +707,10 @@ const ProjectComponentsMapEdit = ({
                         className={classes.formButton}
                         variant="contained"
                         color="primary"
-                        onClick={() =>
-                          saveActionDispatch({ type: "initiateFeatureSave" })
-                        }
-                        // onClick={handleSaveButtonClick}
+                        // onClick={() =>
+                        //   saveActionDispatch({ type: "initiateFeatureSave" })
+                        // }
+                        onClick={handleSaveButtonClick}
```

Comment:
This will be changed back to using the save reducer when drawing in upgraded in the next PR.